apps/services/handlers.py

```diff
@@ -185,14 +198,14 @@ async def create_user_request_review(
         )
         return created_user_request_review
 
+    @staticmethod
     async def read_hostel_accommodation(
-        self,
         *,
         request: Request,
         university_id: int,
         user_request_id: int,
         session: AsyncSession,
-    ):
+    ) -> Optional[HostelAccomodationViewOut]:
```

Comment:
i'm not sure about optional bcs we need to avoid situation when method return None value...

Reply:
It's not the case of this ticket, but ok, we choose return value type. And the same with the router

Reply:
Deleting Optional raise pydantic error, with optional we have jsend return with success and data - null. Wich behaviour is correct?

Reply:
@victor-nuzhniy with Optional it's ok, thanks